test/test.py

```diff
@@ -21,7 +21,7 @@
 
 HTTP_IMAGE_URL = 'http://www.mixedcontentexamples.com/Content/Test/steveholt.jpg'
 HTTPS_IMAGE_URL = 'https://i.imgur.com/3YBkyf6.jpg'
-HTTPS_VIDEO_URL = 'https://dl5.webmfiles.org/big-buck-bunny_trailer.webm'
+HTTPS_VIDEO_URL = 'https://dl6.webmfiles.org/big-buck-bunny_trailer.webm'
```

Comment:
The video test was failing because this URL no longer existed. Seems like it would be better to have these tests files locally but I'm not sure if there was a reason we opted for using public URLs.